frontend/src/services/notification/notification.service.ts

```diff
@@ -0,0 +1,22 @@
+import { toast } from 'react-toastify';
+
+type Message = string | undefined;
+
+class NotificationService {
+    public info(message: Message): void {
+        toast.info(message, { toastId: message });
+    }
+    public success(message: Message): void {
+        toast.success(message, { toastId: message });
+    }
+    public warn(message: Message): void {
+        toast.warn(message, { toastId: message });
+    }
+    public error(message: Message): void {
+        toast.error(message, { toastId: message });
+    }
```

Comment:
create private method show and use it in the public methods

```js
  private show(message: string, parameters: NotificatinOptions): void {
    toast(message, parameters);
  }
```

also create enum for notification type